src/pages/sighting/AnnotationDetail.jsx

```diff
@@ -18,6 +18,8 @@ import useAddKeyword from '../../models/keyword/useAddKeyword';
 import Keywords from './Keywords';
 import MorePhotoMenu from './MorePhotoMenu';
 
+const imageMaxHeight = '80vh';
```

Comment:
This is the first I've seen of viewPort height. Why?

Reply:
well, normally we try to stick to a 4px grid system. but in this case my concern is that the photo will grow to be larger than the viewport. so maybe best to express measurements in terms of the viewport for this?